packages/block-library/src/cover/controls.native.js

```diff
@@ -17,11 +17,12 @@ import {
 	TextControl,
```

Comment:
Same here, there's already a hooked UnitControl in block-editor package, why isn't this one used here instead.